src/components/Chart/index.js

```diff
@@ -55,6 +59,7 @@ export class Chart extends Component {
                 labels,
                 datasets: this.datasets,
             },
+            plugins: plugins || null,
             options: resolveOptions({ type, ...conditions }),
```

Comment:
you should add the plugins variable here, because with the line
`const { type, labels, plugins, ...conditions } = this.props;`
conditions no longer contains plugins